tests/HashMapProperties.hs

```diff
@@ -49,6 +49,14 @@ pFoldable :: [(Int, Int)] -> Bool
 pFoldable = (L.sort . Foldable.foldr (:) []) `eq`
             (L.sort . Foldable.foldr (:) [])
 
+pHashable :: [(Key, Int)] -> [Int] -> Int -> Property
+pHashable xs is salt =
+    x == y ==> hashWithSalt salt x === hashWithSalt salt y
```

Comment:
Surprised this holds if `xs` ever has duplicate keys, as e.g. `fromList [(1,2),(1,3)]` and `fromList [(1,3),(1,2)]` don't result in the same map.